.gitignore

```diff
@@ -16,4 +16,7 @@ bin/
 obj/
 packages/
 
-*.DotSettings.user
\ No newline at end of file
+*.DotSettings.user
+
+# used by test.sh
+/workers/unity/Library-Backup/
```

Comment:
I assume this backup is to avoid "deleting" the Library folder due to the corruption bug we saw? 

If so, are we sure that the bug still exists?

Reply:
I had tested it and it failed inexplicably which could be a sign it still exists, but I'll try a few more times to be sure

Reply:
OK, tested a few more times and it did not explode.

So if it passes CI in this branch then we can close it.

If it fails again then we can reopen UTY-212